docs/stencil-docs/localization/localizing-stores.md

```diff
@@ -88,9 +79,9 @@ Localizing your theme's checkout and order confirmation pages requires customiza
 </div>
 </div>
 
-## Localizing control panel content
+## Localizing your storefront content
 
-The remaining localization options reside outside your theme and are accessible through the BigCommerce control panel:
+Localizing the control panel and options outside your theme and are accessible through the BigCommerce control panel. The [Default Language setting, found in the control panel's Store Profile area](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale), allows localization of the following items:

```

Comment:
```suggestion
The Default Language setting, found in the control panel's [Store Profile](https://login.bigcommerce.com/deep-links/settings/store-profile), allows localization of the following items:

```
I found the first sentence confusing. Can it be removed? I've tried to rewrite it but could not come up with anything.